style.css

```diff
@@ -0,0 +1,69 @@
+body {
+	font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif; font-weight: 300;
+	background-color: lightgray;
+}
+
+a {
+	text-decoration: none;
+	color: black;
+}
+
+/* CLass Modifiers*/
+.student-information {
+	padding: 15px 0px 0px 0px;
+	margin: 0px auto;
```

Comment:
margin/padding short parameters confuse me sometimes. What exactly does this do?